src/lib/holocene/tab/tab.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import { goto } from '$app/navigation';
+  import { getContext } from 'svelte';
+  import type { HTMLButtonAttributes } from 'svelte/elements';
+  import { type TabContext, TABS } from './tabs.svelte';
+
+  interface $$Props extends HTMLButtonAttributes {
+    label: string;
+    id: string;
+    'data-testid'?: string;
+    href?: string;
+    panelId?: string;
+    disabled?: boolean;
+    active?: boolean;
+  }
+
+  export let label: string;
+  export let id: string;
+  export let href = '';
+  export let panelId = null;
```

Comment:
No action item/solution here that I can think of atm... Just wish there was a way we could ensure `panelId`/`tabId` was the same as the `id` in `tab-panel`/`tab` 🤔

Reply:
This was actually caused by importing `goto` in the Tab component. I've refactored it in [e8c35b9](https://github.com/temporalio/ui/pull/1300/commits/e8c35b9883f81a72dfdd2c286dbe9503aa71a417), but using `<svelte:element />` instead of an if/else